### PR TITLE
WIP. [KS] Enforce CompleteRequest() use

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Consistency:
[r50531](https://git.reactos.org/?p=reactos.git;a=commit;h=c6b26fa4525432d8011285c435fab6ee32c0839a) encapsulated IoCompleteRequest(),
[r55983](https://git.reactos.org/?p=reactos.git;a=commit;h=0b301f3da8407eb9667e21b382506a52310821d4) added this new case.

## TODO

- [ ] I don't know how to test this (hack) case.

### CODE REVIEW

**Comment by SergeGautherie** [comment]
As a related PR, who should be listed as Maintainer/Reviewer of drivers/ksfilter[/ks]?

**Comment by BenNottelling** [comment]
> * [ ]  I don't know how to test this (hack) case.

Maybe a testbot would be a good idea then?

**Comment by encodedpr** [comment]
https://github.com/reactos/reactos/blob/893a3c9d030fd8b078cbd747eeefd3f6ce57e560/drivers/base/condrv/dispatch.c#L19

File: drivers/ksfilter/ks/device.c [hunk 1]
```diff
@@ -612,7 +612,7 @@ IKsDevice_Create(
         // ReactOS PnPMgr still sucks
         ASSERT(IoStack->FileObject->FileName.Length == 0);
         Irp->IoStatus.Status = STATUS_SUCCESS;
-        IoCompleteRequest(Irp, IO_NO_INCREMENT);
+        CompleteRequest(Irp, IO_NO_INCREMENT);
```
Comment (encodedpr) [inline drivers/ksfilter/ks/device.c]: no ioinformation structure?

**Comment by encodedpr** [comment]
this change is probably not necessary

File: drivers/ksfilter/ks/device.c (hunk 1 above)
Comment (SergeGautherie) [reply drivers/ksfilter/ks/device.c]: No.

**Comment by learn-more** [comment]
Without testing this it makes no sense to merge it.

**Pull request closed without merging**